### PR TITLE
Mike o multi small changes and messages

### PR DESCRIPTION
**Pull request by MikeOlivas**
- Removed "Home Page" from navbar; now "/" goes to badgein.

- Renamed "User-Station" button to "Edit details" on userManagement page.

- Collapsed "Register new admin" and "Reset admin password" into Admin management dropdown list in navbar.

- Changed (visible) references to registered/unregistered to "online" and "offline" on stationManagement page.

- Removed delete button/column on eventsLog page.

- Changed heading to "Promote existing user to admin" on adminRegister page.

- Added a "success" alert to _layout and removed old message passing in lieu of use connect-flash messages.

### CODE REVIEW

File: views/_layout.njk [hunk 1]
```diff
@@ -87,31 +77,22 @@
...
-          </div>
-        </div> <!-- /alerts container -->
         {% if messages.error %} 
           {% for msg in messages.error%}
           <div class="alert alert-danger alert-dismissible">
             <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
             {{msg}}
           </div>
           {% endfor %}
+        {% elseif messages.success %}
```
Comment (danieleynis) [inline views/_layout.njk]: You beat me to the success messages haha! I implemented the same thing as you have but a little bit differently. The only problem I think I see here is if I want to display both success messages and error messaged. With a `if () else if ()` setup it will allow for only one type to be displayed. To solve this I have implemented it this way on my branch: 

```
      {% if messages.success %} 
          {% for msg in messages.success%}
          <div class="alert alert-success alert-dismissible">
            <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
            {{msg}}
          </div>
          {% endfor %}
        {% endif %}
        {% if messages.error %} 
          {% for msg in messages.error%}
          <div class="alert alert-danger alert-dismissible">
            <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
            {{msg}}
          </div>
          {% endfor %}
        {% endif %}
```

Comment (MikeOlivas) [reply views/_layout.njk]: I didn't think of that! Good call!

File: routes/index.js [hunk 2]
```diff
@@ -585,16 +524,6 @@ function checkRegistration(req, res, next) {
     for (let val in mappedErrors) {
       req.flash('error', mappedErrors[val]['msg']);
     }
-
-    const allData = matchedData(req);
```
Comment (danieleynis) [inline routes/index.js]: Why is this removed? Without it the form will not save fields when there is an error. Also I am pretty sure this does not affect message rendering

Comment (MikeOlivas) [reply routes/index.js]: It definitely wasn't intentional. I must have missed this when I was copy/pasting the stuff that I didn't get from pulling master when I was out of sync. Adding back and resubmitting.
